src/app/pages/2_Sequence_Search.py

```diff
@@ -49,8 +49,12 @@
                 st.error("Please input a search term before you submit")
             else:
                 st.session_state[NCBI_SUMMARY_FORM] = True
-                st.session_state[NCBI_DF] = ncbi.get_data(database, search_term)
-                data_processing.format_ncbi_summary()
+                try:  # if no result is found will go in except block and display error
+                    st.session_state[NCBI_DF] = ncbi.get_data(database, search_term)
+                    data_processing.format_ncbi_summary()
+                except:
```

Comment:
Please initiliaze a logger like [here](https://github.com/cxl-garage/neglected-diagnostics/blob/cc9b9de5e92ff9c5159e11f879277af7abf20095/src/genetic_testing/routers/ncbi.py#L18) and log the error, e.gg
```suggestion
                except e as Exception:
                    logger.error(f"Search error for term: '{search_term}'", exc_info=e)
                
```